Simplified/NYPLCatalogFeedViewController.m

```diff
@@ -82,8 +95,17 @@ + (UIViewController *)navigationFeedWithData:(NYPLXML *)data remoteVC:(NYPLRemot
     } else {
       url = [NSURL URLWithString:gatedXML.attributes[@"restriction-not-met"]];
     }
-    [vc setURL:url];
-    [vc load];
+    if (url != nil) {
+      [vc loadWithURL:url];
+    } else {
+      [NYPLErrorLogger logErrorWithCode:NYPLErrorCodeNoURL
+                                context:NSStringFromClass([self class])
+                                message:@"Server response for age verification lacks a URL to load."
+                               metadata:@{
+                                 @"ageAboveLimit": @(ageAboveLimit),
+                                 @"gateElementXMLAttributes": gatedXML.attributes,
+                               }];
+    }
```

Comment:
What is the user experience if we don't load the URL?

Reply:
also added the Reload UI here. The copy was validated by Risa.